addons/arsenal/functions/fnc_openBox.sqf

```diff
@@ -34,6 +34,8 @@ if (canSuspend) exitWith {
     [{_this call FUNC(openBox)}, _this] call CBA_fnc_directCall;
 };
 
+GVAR(currentBox) = _object;
```

Comment:
Should move below L44 since function might exit without creating display and then `onArsenalClose` won't run to reset the variable to `objNull`.